lib/kernel/src/net_kernel.erl

```diff
@@ -2456,17 +2453,31 @@ hidden_argument() ->
             false
     end.
 
+%%%
+%%% -proto_dist command line argument
+%%%
+
+proto_dist_argument() ->
+    case init:get_argument(proto_dist) of
+        {ok, [Protos | Rest]} ->
```

Comment:
Sidenote: similar functions above do not have that warning.

A follow up question: is there a reason we process `proto_dist` as a list, instead of getting the first item? In other words, is there a case where the user would pass `-proto_dist [proto1, proto2]`. [The docs](https://www.erlang.org/doc/apps/erts/erl_cmd.html#proto_dist) say `-proto_dist Proto`, indicating a single value is expected.